src/GitHub.VisualStudio/Views/GitHubPane/PullRequestFilesView.xaml

```diff
@@ -0,0 +1,138 @@
+﻿<UserControl x:Class="GitHub.VisualStudio.Views.GitHubPane.PullRequestFilesView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:cache="clr-namespace:GitHub.UI.Helpers;assembly=GitHub.UI"
```

Comment:
Could we use `xmlns:ghfvs="https://github.com/github/VisualStudio"` for some of these?

Reply:
Yep, we could!